src/main/java/DukeException.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Implements exception methods for Duke.
+ */
+public class DukeException extends Exception {
+    protected String errorMessage;
```

Comment:
This may not be required since the Exception class has its own String attribute called message as well as a method for its subclasses to retrieve the message.